Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -1466,6 +1476,16 @@ end
 					},
 					Notes = "Returns true if the client has registered to receive messages on the specified plugin channel.",
 				},
+				IsModded =
+				{
+					Returns =
+					{
+						{
+							Type = "boolean",
+						},
+					},
+					Notes = "Returns true if the client is modded with Forge.",
```

Comment:
Forge-specific, or technically, if the "Forge Mod Loader" is installed on the client which is a part of Forge. Won't pick up LiteLoader or other loaders, or other non-loader mods. I'll change IsModded to "IsForgeClient", a clearer description.